printer/kubectl_output_colored_printer.go

```diff
@@ -156,7 +158,7 @@ func (kp *KubectlOutputColoredPrinter) Print(r io.Reader, w io.Writer) {
 	case kubectl.APIVersions:
 		printer = NewTablePrinter(false, kp.DarkBackground, nil) // api-versions always doesn't have header
 
-	case kubectl.Get:
+	case kubectl.Get, kubectl.Events:
```

Comment:
Looks great!

Would like to see a simple test here, just to prevent accidental feature regressions

Reply:
Agreed 👍. I will add a simple test to avoid accidental regressions.